tests/src/integ.rs

```diff
@@ -8,6 +8,14 @@ use std::thread::sleep;
 use std::time::Duration;
 use tempdir::TempDir;
 
+use std::path::Path;
+use std::borrow::Cow;
+
+fn to_canonical_string(path: &Path) -> String {
```

Comment:
I think I'd rather canonicalize the tmpdir earlier.

Right now, the tests all begin with:

```
     let tmpdir = TempDir::new("pazi_integ").unwrap();
     let root = tmpdir.path();
```

I think we could tack on the `canonicalize` after `TmpDir....unwrap()`

If that does work, I think it seems like a cleaner fix since it cleans up the path as early as possible before other paths are joined to it.

Reply:
Good call! This'll be way better.